fluentbit/fluentbit.conf

```diff
@@ -1,3 +1,4 @@
+### cloud-init-output.logs ###
```

Comment:
I considering breaking the different sources up into separate files (using [`@INCLUDE`](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/commands#cmd_include)), but it didn't feel worth it at this stage.

This option could make the code neater if we wanted to make `devx-logs` more configurable (e.g. if we wanted to allow users to opt-out of application log shipping and ship `cloud-init-output.log` only).